core/templates/domain/exploration/InteractionObjectFactory.ts

```diff
@@ -391,10 +391,10 @@ export class InteractionObjectFactory {
   }
 
   convertFromCustomizationArgsBackendDict(
-      interactionId: string | null,
+      interactionId: string,
       caBackendDict: InteractionCustomizationArgsBackendDict
   ): InteractionCustomizationArgs {
-    if (interactionId === null) {
+    if (interactionId === '') {
```

Comment:
Do not use ''. That is not a valid interaction ID.

Reply:
remove this part.